lib/talkable/configuration.rb

```diff
@@ -5,15 +5,20 @@ class Configuration
     attr_accessor :site_slug
     attr_accessor :api_key
     attr_accessor :server
+    attr_accessor :read_timeout
+    attr_accessor :open_timeout
+    attr_accessor :timeout
     attr_accessor :js_integration_library
 
     class UnknownOptionError < StandardError
     end
 
     def initialize
-      self.site_slug  = ENV["TALKABLE_SITE_SLUG"]
-      self.api_key    = ENV["TALKABLE_API_KEY"]
-      self.server     = DEFAULT_SERVER
+      self.site_slug    = ENV["TALKABLE_SITE_SLUG"]
+      self.api_key      = ENV["TALKABLE_API_KEY"]
+      self.server       = DEFAULT_SERVER
+      self.read_timeout = 60
```

Comment:
I vote for a lot shorter timeout. We have 2 (two) seconds in proxies, for example.

Reply:
10 will be ok? Or you suggest less?

Reply:
I’d start with 5.